ansible/roles/tensorrt/tasks/main.yaml

```diff
@@ -4,7 +4,16 @@
     name:
       - libcudnn8={{ cudnn_version }}
       - libcudnn8-dev={{ cudnn_version }}
-      - tensorrt={{ tensorrt_version }}
+      - libnvinfer8={{ tensorrt_version }}
+      - libnvinfer-plugin8={{ tensorrt_version }}
+      - libnvparsers8={{ tensorrt_version }}
+      - libnvonnxparsers8={{ tensorrt_version }}
+      - libnvinfer-bin={{ tensorrt_version }}
+      - libnvinfer-dev={{ tensorrt_version }}
+      - libnvinfer-plugin-dev={{ tensorrt_version }}
+      - libnvparsers-dev={{ tensorrt_version }}
+      - libnvonnxparsers-dev={{ tensorrt_version }}
+      - libnvinfer-samples={{ tensorrt_version }}
```

Comment:
I think you have to update the version in `amd64.env` because `8.4.2.4-1+cuda11.6` is not found for `libnvinfer*`.

Reply:
tensorrt
![image](https://user-images.githubusercontent.com/31987104/185298792-c974000b-ef7e-4646-9a65-850a6915c247.png)

libnvinfer8
![image](https://user-images.githubusercontent.com/31987104/185298845-cb4d36a9-b81f-4434-91a7-04e73e6abed2.png)

Reply:
updated https://github.com/autowarefoundation/autoware/pull/2807/commits/fe986c383e26706dbda81abd8842ce9e4ebe1466